roles/zenith/tasks/main.yml

```diff
@@ -131,3 +131,9 @@
           )
         )
       }}
+
+- name: Migrate services from Consul to CRDs
+  include_tasks: migrate-services.yml
+  when:
+    - consul_server_host is defined
+    - consul_server_port is defined
```

Comment:
Are we happy the clients will keep reconnecting across the auth errors they hit during the transition?